kitsune/dashboards/readouts.py

```diff
@@ -15,7 +15,7 @@
 from django.utils.translation import gettext_lazy as _lazy
 from django.utils.translation import pgettext_lazy
 from django.utils.translation import ugettext as _
-from jinja2 import Markup
+from markupsafe import Markup
```

Comment:
Both `Markup` and `escape` are no longer available from `jinja2`. They must be imported directly from the `markupsafe` package.